__tests__/index.test.js

```diff
@@ -258,3 +260,60 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("PATCH 200: responds with a 200 code when increasing the vote count of a certain article_id", () => {
+    const newVote = 5;
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: newVote })
+      .expect(200)
+      .then(({ body }) => {
+        const { votes } = body;
+        expect(votes.votes).toBe(105);
+      });
+  });
+    test("PATCH 200: responds with a 200 code when decreasing the vote count of a certain article_id", () => {
+      const newVote = -5;
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: newVote })
+        .expect(200)
+        .then(({ body }) => {
+          const { votes } = body;
+          expect(votes.votes).toBe(95);
+        });
+    });
+  test("404: receive a 404 error when article id is not found", () => {
+    const newVote = 5;
+    return request(app)
+      .patch("/api/articles/999")
+      .send({ inc_votes: newVote })
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("Not found");
+      });
+  });
+  test("404: receive a 404 error when newVote isn't a number", () => {
+    const newVote = 'hello';
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: newVote })
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("Not found");
+      });
+  });
+  test("PATCH 400 responds with an appropriate error message when provided with an incorrect end point", () => {
+    const newVote = 5;
+    return request(app)
+      .patch("/api/articles/articlesid")
+      .send({ inc_votes: newVote })
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("Bad request");
+      });
+  });
+});
```

Comment:
Could potentially add another test: 200 | If someone has missed their inc_votes key, the article returns unchanged.

^^^this is optional 👍